src/main/java/com/st/utopia/cancellationservice/model/Ticket.java

```diff
@@ -0,0 +1,191 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * An Ticket model.
+ *
+ * @author Henry Cho
```

Comment:
`@author` tags (and other Javadoc tags) should generally be the *last* things in a Javadoc comment, not the (almost) *first* things.